### PR TITLE
CIN banner

### PR DESCRIPTION
**Pull request by andre-pickering**
## Pull request overview

Give a general description of why a change is being made, include issue number(s) being fixed if relevant

## Pull request checklist

Please check if your PR fulfils the following:
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Tests have been run locally and are passing (`run_tests_locally()`)
- [ ] Code is styled according to tidyverse styling (checked locally with `tidy_code()`)

## What is the current behaviour?

Please describe the current behaviour that you are modifying, or link to a relevant issue. Include screenshots for UI changes where possible.

## What is the new behaviour?

Please describe the behaviour or changes that are being added by this PR. Include screenshots for UI changes where possible.

## Anything else

Add any notes for people reviewing and testing your code that are appropriate. Tag a @person to review if someone in particular needs to see this.

### CODE REVIEW

**Comment by yauemily** [comment]
@rmbielby @chfoster For some reason, we can't update any of the tests. Can anyone on the EES side update the tests please?

File: ui.R [hunk 1]
```diff
@@ -69,9 +69,7 @@ ui <- function(input, output, session) {
     cookies_banner_ui(
       name = "Children's Social Care - Outcomes and Enablers"
     ),
-    dfeshiny::custom_disconnect_message(
-      links = c(site_primary, site_overflow)
-    ),
+    customDisconnectMessage(),
```
Comment (chfoster) [inline ui.R:72]: Removing dfeshiny:: from here will be the reason for the error seen below: 
image

Comment (chfoster) [reply ui.R:72]: @yauemily I'm wrong actually - what was the reason for changing this? dfeshiny documentation shows that the old way was right?? https://dfe-analytical-services.github.io/dfeshiny/reference/custom_disconnect_message.html

Comment (yauemily) [reply ui.R:72]: @chfoster I have no idea. I think it was Rich, who changed it

**Review by yauemily — APPROVED** [review]

**Pull request merged**